### ISSUE TITLE
behat.paths not included in command output

### ISSUE THREAD

**Issue by bobbygryzynger** [acquia/blt#2501]
My system information:
* Operating system type: MacOS
* Operating system version: 10.13.2
* BLT version: 8.9.10

Output of `blt doctor`:
```+----------------------+---------------------------------------------------------------------------------+
| Property             | Value                                                                           |
+----------------------+---------------------------------------------------------------------------------+
| %paths               |                                                                                 |
|  - %root             | /var/www/<site>/docroot                                                         |
|  - %site             | sites/default                                                                   |
|  - %modules          | modules                                                                         |
|  - %themes           | themes                                                                          |
|  - %config-sync      | /var/www/<site>/config/default                                                  |
|  - %files            | sites/default/files                                                             |
|  - %private          | /var/www/<site>/files-private                                                   |
|  - %temp             | /tmp                                                                            |
| admin-theme          | seven                                                                           |
| blt-version          | 8.9.10                                                                          |
| bootstrap            | Successful                                                                      |
| config-sync          | /var/www/<site>/config/default                                                  |
| db-driver            | mysql                                                                           |
| db-hostname          | localhost                                                                       |
| db-name              | <site>                                                                          |
| db-password          | admin                                                                           |
| db-port              | 3306                                                                            |
| db-status            | Connected                                                                       |
| db-username          | admin                                                                           |
| dev-desktop-enabled  | false                                                                           |
| drupal-settings-file | sites/default/settings.php                                                      |
| drupal-version       | 8.4.4                                                                           |
| drupal-vm-enabled    | true                                                                            |
| drush-alias-files    | /var/www/<site>/docroot/../drush/site-aliases/example.local.aliases.drushrc.php |
|                      | /var/www/<site>/docroot/../drush/site-aliases/aliases.drushrc.php               |
|                      | /var/www/<site>/docroot/../drush/site-aliases/example.acsf.aliases.drushrc.php  |
| drush-conf           | /var/www/<site>/drush/drushrc.php                                               |
| drush-script         | /var/www/<site>/vendor/drush/drush/drush.php                                    |
| drush-temp           | /tmp                                                                            |
| drush-version        | 8.1.15                                                                          |
| files                | sites/default/files                                                             |
| install-profile      | standard                                                                        |
| local-drushrc        | /var/www/<site>/docroot/sites/default/local.drushrc.php                         |
| local-settings       | /var/www/<site>/docroot/sites/default/settings/local.settings.php               |
| modules              | modules                                                                         |
| php-bin              | /usr/bin/php                                                                    |
| php-conf             | /etc/php/5.6/cli/php.ini                                                        |
| php-mysql            | /var/run/mysqld/mysqld.sock                                                     |
| php-os               | Linux                                                                           |
| private              | /var/www/<site>/files-private                                                   |
| root                 | /var/www/<site>/docroot                                                         |
| shell                |                                                                                 |
| site                 | sites/default                                                                   |
| temp                 | /tmp                                                                            |
| theme                | nyunursing                                                                      |
| themes               | themes                                                                          |
| uri                  | http://<site>.local                                                             |
| user                 |                                                                                 |
+----------------------+---------------------------------------------------------------------------------+
+--------------------------+----------------------------------------------------------------------------------+
| Check                    | Outcome                                                                          |
+--------------------------+----------------------------------------------------------------------------------+
| checkDocrootExists       | Found docroot.                                                                   |
| checkCoreExists          | Drupal core exists                                                               |
| checkSettingsFile        | BLT settings are included in settings file.                                      |
| checkLocalSettingsFile   | Found your local settings file.                                                  |
| checkLocalDrushFile      | Found your local drush settings file.                                            |
| checkUri                 | $options['uri'] is set.                                                          |
| checkUriResponse         | Received a response from site http://<site>.local.                               |
| checkFileSystem:%files   | Public files directory is writable.                                              |
| checkFileSystem:%private | Private files directory is writable.                                             |
| checkFileSystem:%temp    | Temporary files directory is writable.                                           |
| checkDbConnection        | Connected to database.                                                           |
| checkDrupalBootstrapped  | Bootstrapped Drupal via drush.                                                   |
| checkDrupalInstalled     | Drupal is installed.                                                             |
| checkCaching:page        | Drupal cache is disabled.                                                        |
| checkCaching:css         | CSS preprocessing is disabled.                                                   |
| checkCaching:js          | JS preprocessing is disabled.                                                    |
| checkComposer:require    | acquia/blt is in composer.json's require object.                                 |
| checkBehat:exists        | Behat local settings file exists.                                                |
| checkBehat:root          | Behat drupal_root is set correctly for Drupal VM.                                |
| checkProjectYml:keys     | project.yml has no deprecated keys.                                              |
| checkDrupalVm:alias      | drush.aliases.local exists your drush aliases file.                              |
| checkDrupalVm            | Drupal VM is configured correctly.                                               |
| checkSimpleSamlPhp       | /var/www/<site>/docroot/../vendor/simplesamlphp/simplesamlphp/metadata/saml20-id |
|                          | p-remote.php is missing. Run `blt simplesamlphp:build:config`.                   |
| checkPhpDateTimezone     | PHP setting for date.timezone is correctly set                                   |
+--------------------------+----------------------------------------------------------------------------------+
```

When I run this command:
```
$ blt tests:behat -D behat.paths=${PWD}/tests/behat/features/my.feature
```

I get the following output:
```
$ blt tests:behat -D behat.paths=${PWD}/tests/behat/features/my.feature
[warning] The xDebug extension is loaded. This will significantly decrease performance.
Executing command blt 'tests:behat' --define=drush.alias=self inside of Drupal VM...
[warning] The xDebug extension is loaded. This will significantly decrease performance.
[Robo\Common\ProcessExecutor] Running 'google-chrome' --headless --disable-gpu --remote-debugging-port=9222  https://www.chromestatus.com --disable-web-security --user-data-dir > /dev/null 2>&1 in /var/www/<site>
[Testing\Behat] Running behat  --format pretty /var/www/<site>/tests/behat --colors --no-interaction --stop-on-failure --strict --config /var/www/<site>/tests/behat/<site>-behat.yml --profile default --tags '~ajax&&~experimental&&~lightningextension' -v
```

And I expected this to happen:
The `behat.paths` value to be included as part of the generated command as indicated in the [project documentation](http://blt.readthedocs.io/en/latest/readme/testing/#behat).

**Comment by grasmash** [comment]
I think this is a bug in the redispatch to the VM. If you ssh into the VM, it should work.

**Comment by bobbygryzynger** [comment]
@grasmash yes, I can confirm that `ssh`-ed into the VM the correct feature is run.

**Comment by grasmash** [comment]
https://github.com/acquia/blt/pull/2503

**Comment by bobbygryzynger** [comment]
I couldn't apply this as a patch to `8.9.10` so I'll have to look at upgrading at a later date.

```
  - Applying patches for acquia/blt
    https://patch-diff.githubusercontent.com/raw/acquia/blt/pull/2503.patch (Fixes #2501: behat.paths not included in command output.)
   Could not apply patch! Skipping. The error was: Cannot apply patch https://patch-diff.githubusercontent.com/raw/acquia/blt/pull/2503.patch

```
